### PR TITLE
Minor net command fixes

### PR DESCRIPTION
**Pull request by LJSonik**
A few minor net command-related fixes.

### CODE REVIEW

File: src/d_clisrv.c [hunk 1]
```diff
@@ -3292,7 +3292,7 @@ void SV_StopServer(void)
 	localtextcmd[0] = 0;
 	localtextcmd2[0] = 0;
 
-	for (i = 0; i < BACKUPTICS; i++)
+	for (i = firstticstosend; i < firstticstosend + BACKUPTICS; i++)
 		D_Clearticcmd(i);
```
Comment (MonsterIestyn) [inline src/d_clisrv.c]: Should note, what you're doing here could equally be achieved by having instead the line below be "D_Clearticcmd(firstticstosend+i);"

**Comment by MonsterIestyn** [comment]
Other than my comment above (which isn't really much of a problem anyway), all seems fine to me

File: src/d_clisrv.c [hunk 2]
```diff
@@ -401,8 +401,7 @@ static void ExtraDataTicker(void)
 							DEBFILE(va("player %d kicked [gametic=%u] reason as follows:\n", i, gametic));
 						}
 						CONS_Alert(CONS_WARNING, M_GetText("Got unknown net command [%s]=%d (max %d)\n"), sizeu1(curpos - bufferstart), *curpos, bufferstart[0]);
-						D_FreeTextcmd(gametic);
-						return;
+						break;
```
Comment (toasterone) [inline src/d_clisrv.c]: So walk me through this change. Why's it what we want to do here?

**Comment by toasterone** [comment]
👍

**Comment by MonsterIestyn** [comment]
Hi LJ Sonic, could you update this to the current state of master so perhaps I can merge it at some point?

**Comment by wolfy852** [comment]
:+1:

**Comment by alama** [comment]
:+1:

**Pull request merged**